packages/shared/src/components/Feed.tsx

```diff
@@ -132,6 +137,7 @@ export default function Feed<T>({
   variables,
   className,
   header,
+  isFeedSquad,
```

Comment:
Remember the main feed can contain both normal articles as well as share articles.
Think the evaluation has to be per item?

Reply:
I think we should use the post type

Reply:
Oh, in that case, we should then use `post.type`.